managed-databases/mongodb/how-to/connect-database-instance.mdx

```diff
@@ -0,0 +1,271 @@
+---
+meta:
+  title: How to connect to a MongoDB™ Database Instance
+  description: This page explains how to connect to a MongoDB™ Database Instance
+content:
+  h1: How to connect to a MongoDB™ Database Instance
+  paragraph: This page explains how to connect to a MongoDB™ Database Instance
+tags: mongodb mongodb document database-instance managed-database database
+dates:
+  validation: 2024-09-18
+  posted: 2024-09-18
+categories:
+  - managed-databases
+---
+
+<Macro id="requirements" />
+
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```

Comment:
If it's in Private Beta, do they also need a quota / have signed up to the beta? (Same comment for other files)

Reply:
@scw-wtimmermans how will the private beta work?